### PR TITLE
Deployment pipeline to deploy bts javascore to Testnet#342

### PR DESCRIPTION
**Pull request by dwdraju**

### CODE REVIEW

File: .github/workflows/deploy-bts-javascore.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+      - name: Deploy javascore bts optimized jar
+        working-directory: ./javascore
+        shell: bash
+        timeout-minutes: 1
+        env:
+          GOLOOP_RPC_KEY_PASSWORD: ${{ secrets.GOLOOP_RPC_KEY_PASSWORD }}
+          GOLOOP_RPC_URI: ${{ secrets.GOLOOP_RPC_URI }}
+        run: |
+          goloop rpc sendtx deploy bts/build/libs/bts-optimized.jar \
```
Comment (CyrusVorwald-ICON) [inline .github/workflows/deploy-bts-javascore.yml:56]: I do not think that these param values should be hard coded

**Comment by CyrusVorwald-ICON** [comment]
> @dwdraju Could you implement this script in general way, not testnet specific? Provide choose net for executor. You can take a look my implementation in the realese_tag_manager script.

I agree, see my comment here https://github.com/icon-project/icon-bridge/pull/346/files#r955906262

File: .github/workflows/deploy-bts-javascore.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
+name: 'Deploy bts Javascore'
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - deploy-bts-javascore-#342
```
Comment (andrii-kl) [inline .github/workflows/deploy-bts-javascore.yml:7]: remove this test branch

**Comment by andrii-kl** [comment]
I would merge _Deploy bts Javascore_ and  _Deploy bmc Javascore_ in to one github action, with ability of choosing what to deploy,  but it's not critical.

**Comment by dwdraju** [comment]
Added unified script through [this PR](https://github.com/icon-project/icon-bridge/pull/384), so this PR is no longer valid.

**Pull request closed without merging**